appuio/haproxy/Dockerfile

```diff
@@ -0,0 +1,7 @@
+from haproxytech/haproxy-debian:2.7.3
```

Comment:
Please use the latest version (currently 2.8.3)

Reply:
done, also pushed